docs/conf.py

```diff
@@ -1,3 +1,3 @@
 from crate.theme.rtd.conf.crate_howtos import *
 
-exclude_patterns = ["out/**"]
+exclude_patterns = ["out/**", "README.rst"]
```

Comment:
minor, optional: this can be left in. once this build system is upgraded to use the docs utils this entire line will be removed. for the time being, it's harmless